cmd/flags.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import "flag"
+
+var configFlags = []*flag.Flag{
+	loglevelFlag,
+	nodeConfigFlag,
+	trustedHashFlag,
+	trustedPeerFlag,
+	coreRemoteFlag,
+}
+
+var (
+	loglevelFlag = &flag.Flag{
+		Name:     "log.level",
+		Usage:    "DEBUG, INFO, WARN, ERROR, DPANIC, PANIC, FATAL, and\n// their lower-case forms",
+		DefValue: "info",
```

Comment:
We should stop turning logging by default IMO and make empty DefValue. Can you pls include this in this PR?

Reply:
wait why? Log info level should always be default?

Reply:
> Log info level should always be default?

Exactly, why? IMHO logging is mostly for power users. I don't have a strong opinion though on such minor things for now, but for light clients, I would remove logging by default.

Reply:
No I think that's not good. I don't know any other client software that disables logging by default. If the user doesn't want to see the logs, they can run it as a background proc.

Reply:
Well, how often do you encounter logs while working with any software clients daily? Anyway, I think keeping logging by default is good for now, as things will break and triers will be able to report logs.

Reply:
We also should pipe those to some file by default mb

Reply:
https://github.com/celestiaorg/celestia-node/issues/245